GruntJS to compile LESS

juanbrujo opened the pull request:
now using GruntJS to compile LESS.
modules:
- livereload
- grunt
- grunt-contrib-less
- grunt-contrib-watch
- grunt-newer

juanbrujo committed:
GruntJS integration

new build through GruntJS

juanbrujo committed:
/src route modified

juanbrujo committed:
Better README

added BUILD flow to README.md

juanbrujo committed:
not yet

asciimoo commented:
Why do you want to delete the makefile/css files?
They both may come in handy if don't want to install nodejs/grunt/less to get only the css files

rafaelrinaldi commented:
There's no point on replacing the current Makefile (who already does the job really well) for Grunt.

juanbrujo commented:
@asciimoo just because I asked you so, to transfer the .less compilation/minification to GruntJS instead using the makefile. The CSS files were deleted because they're generated when GruntJS is running. It's up to you, I'm working this way the animated transitions improvement.

asciimoo commented:
It's nice if we have the gruntfile as an option, but i agree with @rafaelrinaldi about keeping the makefile/css

juanbrujo commented:
so, it's a merge not a replacement.

asciimoo commented:
yes

juanbrujo committed:
Animated transitions added

- added animated transitions to charts using data-cp-transition and
data-cp-delay attributes

juanbrujo committed:
Merge branch 'master' of https://github.com/juanbrujo/cssplot

Conflicts:
	demo.html

juanbrujo committed:
created cssplot.animations.less

juanbrujo committed:
added instructions for animated charts

juanbrujo committed:
better readme
